src/main/resources/application.yaml

```diff
@@ -51,8 +51,8 @@ spring:
 
 monitoring:
   incomplete-envelopes:
-    cron: 0 0 7 ? * MON-FRI # can be discussed later
-    enabled: false
+    cron: ${INCOMPLETE_ENVELOPES_CRON:0 0 7 ? * MON-FRI} # can be discussed later
```

Comment:
Do we need the defaults here ?

Reply:
i would prefer, same as reporting config couple lines below